docs-new/docs/graphql/core/event-triggers/create-trigger.mdx

```diff
@@ -1,64 +1,49 @@
-<div class="meta" description="Create an event trigger with Hasura"
-keywords="hasura, docs, event trigger, create">
-
-</div>
+---
+  description: Create an event trigger with Hasura
+  keywords:
+    - hasura
+    - docs
+    - event trigger
+    - create
+---
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import Thumbnail from "@site/src/components/Thumbnail/Thumbnail";
+import TOCInline from '@theme/TOCInline';
 
 # Creating an event trigger
 
-<div class="contents" backlinks="none" depth="2" local="">
-
-Table of contents
-
-</div>
-
 ## Introduction
 
-Event triggers can be created using the Hasura console, CLI or metadata
-APIs.
-
-<div class="rst-class">
-
-api_tabs
-
-</div>
-
-<div class="tabs">
-
-<div class="tab">
-
-Console
-
-Open the Hasura console, head to the `Events` tab and click on the
-`Create` button to open the page below:
-
-<div class="thumbnail" alt="Create an event trigger">
+Event triggers can be created using the Hasura console, CLI or metadata APIs.
 
-/img/graphql/core/event-triggers/create-event-trigger.png
+<Tabs className="api-tabs">
+<TabItem value="console" label="Console">
 
-</div>
+Open the Hasura console, head to the `Events` tab and click on the `Create` button to open the page below:
+<Thumbnail
+src="/img/graphql/core/event-triggers/create-event-trigger.png"
+alt="Create an event trigger"
+/>
 
-</div>
+</TabItem>
+<TabItem value="cli" label="CLI">
 
-<div class="tab">
-
-CLI
-
-You can add an event triggers in the `tables.yaml` file inside the
-`metadata` directory:
+   You can add an event triggers in the ``tables.yaml`` file inside the ``metadata`` directory:
```

Comment:
Inline code can be with single backticks in markdown. In RST a single backtick represent link and hence the double backtick usage. Also, when markdown is invloved, it is better to start at beginning of line.
```suggestion
You can add an event triggers in the `tables.yaml` file inside the `metadata` directory:
```